src/config.js

```diff
@@ -13,8 +13,8 @@ const _ASSURANCE_STRICT = false
 
 export const CARDANO_CONFIG = {
   TESTNET: {
-    PROTOCOL_MAGIC: 633343913,
-    API_ROOT: 'https://stg-yoroi-backend.yoroiwallet.com/api',
+    PROTOCOL_MAGIC: 1097911063,
+    API_ROOT: 'https://testnet-yoroi-backend.yoroiwallet.com/api',
     EXPLORER_URL_FOR_TX: (tx: string) => `https://explorer.iohkdev.io/tx/${tx}`,
```

Comment:
we need to update this one to: `https://cardano-explorer.cardano-testnet.iohkdev.io/`